components/alert/AlertMap.vue

```diff
@@ -222,103 +273,127 @@ export default Vue.extend({
-      this.map = (<LMap>this.$refs["map"]).mapObject as L.Map;
-      this.remaining = (<LControl>this.$refs["timer"]);
-      this.map.attributionControl.addAttribution('Tiles extracted from <a title="Planetside 2® Public Test Server" href="https://forums.daybreakgames.com/ps2/index.php?threads/read-first-test-server-policies-download-link.114038/">PTS client</a>')
-      this.map.attributionControl.addAttribution('Hex and region data from <a href="https://census.daybreakgames.com">Census</a>');
-      this.map.attributionControl.addAttribution('Oshur region data from <a title="Planetside 2 API developers\' Discord channel on the unofficial Planetside 2 Discord server" href="https://discord.com/channels/251073753759481856/451032574538547201">#api-dev</a>');
-      this.map.attributionControl.addAttribution('Oshur hex data from RiderAnton');
-      this.map.attributionControl.getContainer()?.classList.add('alert-map-attribution');
-      this.map.createPane("hexPane", this.map.getPane("overlayPane"));
-      this.map.createPane("linkPane", this.map.getPane("overlayPane"));
-      this.map.createPane("badgePane", this.map.getPane("overlayPane"));
-      this.map.createPane("badgeTextPane", this.map.getPane("overlayPane"));
+      this.map = (this.$refs.map as LMap).mapObject as L.Map
+      this.remaining = this.$refs.timer as LControl
+      this.map.attributionControl.addAttribution(
+        'Tiles from <a title="Planetside 2® Public Test Server" href="https://forums.daybreakgames.com/ps2/index.php?threads/read-first-test-server-policies-download-link.114038/">PTS client</a>'
+      )
+      this.map.attributionControl.addAttribution(
+        'Map data from <a href="https://census.daybreakgames.com">Census</a>'
+      )
+      this.map.attributionControl.addAttribution(
+        'Oshur data from <a title="Planetside 2 API developers\' Discord channel on the unofficial Planetside 2 Discord server" href="https://discord.com/channels/251073753759481856/451032574538547201">#api-dev</a>'
+      )
+      this.map.attributionControl.addAttribution('Oshur hexes from RiderAnton')
+      this.map.attributionControl
+        .getContainer()
+        ?.classList.add('whitespace-nowrap')
+      this.map.createPane('hexPane', this.map.getPane('overlayPane'))
+      this.map.createPane('linkPane', this.map.getPane('overlayPane'))
+      this.map.createPane('badgePane', this.map.getPane('overlayPane'))
+      this.map.createPane('badgeTextPane', this.map.getPane('overlayPane'))
       this.map.on('drag', () => {
-        this.map.fitBounds(this.map.getBounds());
-      });
-      this.$L.tileLayer(this.url, {
-        bounds: [[0, 0], [-256, 256]],
-        minNativeZoom: 2,
-        maxNativeZoom: 5
-      }).addTo(this.map);
-
-      this.zoomInSound.loop = false;
-      this.zoomOutSound.loop = false;
-      this.zoomInSound.volume = 0.1;
-      this.zoomOutSound.volume = 0.1;
+        this.map.fitBounds(this.map.getBounds())
+      })
+      this.$L
```

Comment:
Could we change this to `this.mapLayer` so it better denotes what it is?

Reply:
Which part are you talking about? `this.$L -> this.mapLayer` or `this.map -> this.mapLayer`? Or something else?

Reply:
or `this.$L.tileLayer -> this.$L.mapLayer`?

Reply:
Because `this.$L` is the Leaflet object controlled by nuxt-leaflet/vue2-leaflet so modifying its name or any of its functions would be quite hacky

Reply:
Ah if it's plugin releated I'll resolve it